File: Data Structures & Algorithms/counting-bits/submission-0.cpp
class Solution {
public:
    vector<int> countBits(int n) {
        vector<int> ans;
        for(int i=0;i<=n;i++){
            if(i==0 || i == 1) ans.push_back(i);
            else if(i%2 == 0) ans.push_back(ans[i/2]);
            else ans.push_back(ans[i/2] + 1);
        }
        return ans;
    }
};

File: Data Structures & Algorithms/merge-sorted-array/submission-0.java
class Solution {
    public void merge(int[] nums1, int m, int[] nums2, int n) {
        int[] ans = new int[m+n];
        int x = 0;
        int i=0, j = 0;
        while(i < m && j < n) {
            if(nums1[i] < nums2[j]){
                ans[x++] = nums1[i];
                i++;
            } else{
                ans[x++] = nums2[j];
                j++;
            }
        }
        while(i < m){
            ans[x++] = nums1[i++];
        }
        while(j < n) {
            ans[x++] = nums2[j++];
        }
        for(int t=0;t<m+n;t++) {
            nums1[t] = ans[t];
        }
    }
}